arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -1821,6 +1743,13 @@
             "$ref": "#/definitions/DeepCreatedOrigin"
           }
         },
+        "geoFilters": {
```

Comment:
is geoFilters updatable ? From the description it looks like it is. If so then this should also be present in EndpointPropertiesUpdateParameters.